### ISSUE TITLE
clarification on setting session using token

### ISSUE THREAD

**Issue by johndpope** [supabase-community/supabase-py#306]
related 
https://github.com/supabase-community/supabase-swift/issues/49

**Comment by luquitared** [comment]
there is an AuthClient class that has a method called get_client_with_access_token() but like you I am not sure how to instantiate this and use it. This repo would benefit greatly with an example of setting the session using the token in the readme or the fastAPI example folder.

+1 on this!!

**Comment by johndpope** [comment]
related - https://github.com/supabase-community/supabase-swift/issues/4
you may be able to cherry pick swift example

**Comment by J0** [comment]
Agreed that we should probably try to do a FastAPI Auth tutorial sometime. Going to move this to `gotrue-py` for better tracking
